target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -238,6 +238,7 @@ get_status_led() {
 		status_led="$board:amber:power"
 		;;
 	whr-g300n|\
+	zbt-we2026|\
```

Comment:
Is there a chance to use a not red LED here. I really don't like to use red LEDs in case there is no error/issue.

I couldn't find any documentation about or even a high res picture of the board. No idea whether there is a green power led as well.